src/nmod_mpoly/mpolyun.c

```diff
@@ -404,301 +404,6 @@ int nmod_mpolyun_equal(
     return 1;
 }
 
-
```

Comment:
I'm a bit uneasy about outright deleting this kind of ``mpoly`` code. It sometimes helps understand the (poorly documented) code that is actually used. Could you ifdef it out instead, with a comment like "This code is unused. TODO: delete it when the surrounding code is adequately tested and commented."? I think I've already done this elsewhere.